docs/how-to-migrate-from-smartlock.md

```diff
@@ -0,0 +1,791 @@
+# Migrating from the Google Smart Lock for Passwords API to OpenYOLO
+
+This document describes how to migrate your current integration with the
+Google Smart Lock for Passwords to the OpenYOLO API. If your application does
+not currently utilize Smart Lock for Passwords, we recommend instead reading
+the main [README](../README.md), which contains a getting started guide and
+links to other resources.
+
+Google will continue to offer the Smart Lock for Passwords API for the
```

Comment:
IMO we should draw more attention to this to help avoid it being overlooked. Second sentence in the second paragraph is prime skim over territory. What do you think about a new section "Target audience" with the first sentence being "In 2018, the Smart Lock...." in bold?

Reply:
I'll move this to the first paragraph.